File: fix-import.js
const fs = require('fs');
const file = 'src/app/api/products/[id]/route.ts';
let content = fs.readFileSync(file, 'utf8');

if (!content.includes('revalidatePath }')) {
    content = 'import { revalidatePath } from "next/cache";\n' + content;
    fs.writeFileSync(file, content, 'utf8');
    console.log('Fixed import');
}

File: fix_import.py
with open("src/app/api/analytics/dashboard/route.ts", "r", encoding="utf-8") as f:
    content = f.read()

content = content.replace('import mongoose from "mongoose";\\n    const abandonedCollections', 'const abandonedCollections')
content = 'import mongoose from "mongoose";\n' + content

with open("src/app/api/analytics/dashboard/route.ts", "w", encoding="utf-8") as f:
    f.write(content)
print("Fixed mongoose import")
